Validation/RecoEgamma/python/ElectronMcFakePostValidator_cfi.py

```diff
@@ -8,7 +8,7 @@
 
 electronMcFakePostValidator = DQMEDHarvester("ElectronMcFakePostValidator",
 
-  Verbosity = cms.untracked.int32(0),
+  Verbosity = cms.untracked.int32(1),
```

Comment:
This change is supposed to be printing the ME, from:
https://github.com/cms-sw/cmssw/blob/2db55fa80748eeb2ea48abd580a15631e3d7811b/DQMOffline/EGamma/src/ElectronDqmHarvesterBase.cc#L377-L378
It is not desired to flood the logs, I guess you forgot to disable it for debugging perhaps?
Thanks

Reply:
heuh, yes. sorry.
do you want I make the change and repost the PR?

Reply:
Yes please.
And if at the same time you can certify that changes seen are inline with what you expect:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_0_X_2021-07-01-2300+78a26f/43793/dqm-histo-comparison-summary.html
I believe they are but small stats confuse me and I am not an expert on EGamma. Thank you